semantic-kernel/concepts/text-search/index.md

```diff
@@ -200,9 +200,88 @@ Next we recommend looking at [Text Search Abstractions](./text-search-abstractio
 ::: zone-end
 ::: zone pivot="programming-language-python"
 
-## Coming soon
+## Implementing RAG using web text search
 
-More coming soon.
+In the following sample code, you can choose between using Bing or Google to perform web search operations.
+
+> [!TIP]
+> Install the required packages using:
+>
+> `pip install semantic-kernel`
+> `pip install semantic-kernel-plugins-web`
+
+### Create text search instance
+
+Each sample creates a text search instance and then performs a search operation to get results for the provided query.
+
+#### Bing web search
+
+```python
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+from semantic_kernel.plugins.web import BingTextSearch
+from semantic_kernel.functions import KernelArguments
+
+# Create a kernel with OpenAI chat completion
+kernel = Kernel()
+kernel.add_service(
+    OpenAIChatCompletion(
+        model_id="gpt-4o",
```

Comment:
this is not a valid param, should be ai_model_id or left off and picked from the env variables